managers/CpbSprintManager.cpp

```diff
@@ -65,6 +65,36 @@ void SprintManager::createSprint()
     newSprint->deleteLater();
 }
 
+void SprintManager::createSprint(const QString& sprintName, const QString& startDate, const QString& endDate)
+{
+    Sprint* newSprint = new Sprint(sprintName,
+                                   QDate::fromString(startDate, "dd.MM.yyyy"),
```

Comment:
we need to move such strings "dd.MM.yyyy" to some constants... but now let's keep it under unnamed namespace